app/src/main/java/org/fossasia/susi/ai/chat/ChatActivity.kt

```diff
@@ -207,7 +208,7 @@ class ChatActivity : AppCompatActivity(), IChatView {
             } else {
                 if (bottom < oldBottom) {
                     rv_chat_feed.postDelayed({
-                        var scrollTo = rv_chat_feed.adapter.itemCount - 1
+                        var scrollTo = (rv_chat_feed.adapter as ChatFeedRecyclerAdapter).itemCount - 1
```

Comment:
Don't do casts

Reply:
But it shows another error smart cast to 'ChatFeedRecyclerAdapter' is impossible, because 'rv_chat_feed.adapter' is a complex expression

Reply:
Cast is not actually needed because itemCount is present in generic adapter, and if it is not, add an if statement wrapping the cast